src/frontend/src/flows/manage/settingsDropdown.ts

```diff
@@ -1,22 +1,42 @@
 import { dropdownIcon } from "$src/components/icons";
+import { handleKeyPress } from "$src/utils/keyboard.utils";
+import { Chan } from "$src/utils/utils";
 import { html, TemplateResult } from "lit-html";
+import { asyncReplace } from "lit-html/directives/async-replace.js";
 
-export const settingsDropdown = ({
-  alias,
-  id,
-  settings,
-}: {
+interface SettingsDropdown {
   alias: string;
   id: string;
   settings: { fn: () => void; caption: string; action: string }[];
-}): TemplateResult => {
+}
+
+const mkChasm = ({ alias, id, settings }: SettingsDropdown): TemplateResult => {
```

Comment:
`mkChasm` is already used somewhere else, plus this is not really a chasm but a dropdown. Would it make sense to use a single function, `settingsDropdown`? What is the reason for `settingsDropdown = mkChasm`?